core/src/main/java/org/elasticsearch/search/lookup/LeafDocLookup.java

```diff
@@ -68,13 +68,11 @@ public void setDocument(int docId) {
 
     @Override
     public ScriptDocValues<?> get(Object key) {
```

Comment:
Not if I want to implement the Map interface. We could stop implementing Map, at least in master.  Which is a pretty good idea, I think.